src/azul/indexer/index_controller.py

```diff
@@ -274,6 +292,20 @@ def aggregate(self, event: Iterable[SQSRecord], *, retry=False):
             log.warning('Failed to aggregate tallies: %r', tallies_by_entity.values(), exc_info=True)
             raise
 
+    def redrive(self, event):
+        for record in event:
+            message = json.loads(record.body)
+            receive_count = record.to_dict()['attributes']['ApproximateReceiveCount']
+            attempts = 1 if int(receive_count) > 1 else 0 + message['attempts']
```

Comment:
```suggestion
            attempts = 1 if int(receive_count) > 1 else message['attempts']
```
or 
```suggestion
            attempts = (1 if int(receive_count) > 1 else 0) + message['attempts']
```
depending on intent. as written the `0 +` has no effect.

Reply:
In the latter case, consider
```suggestion
            attempts = message['attempts']
            if int(recieve_count) > 1:
                attempts += 1
```
I know it's more lines but personally I find it easier to follow. Your call.